packages/training/python/tests/test_enhanced_rewards.py

```
@@ -0,0 +1,679 @@
+
+
+# =============================================================================
+# Temporal Credit Tests
+# =============================================================================
+
+class TestTemporalCredit:
+    """Tests for temporal credit assignment."""
+    
+    def test_is_trading_action(self):
+        """Trading actions are correctly identified."""
+        assert is_trading_action("buy") == True
```

Ruff (0.14.11)

Avoid equality comparisons to `True`; use `is_trading_action("buy"):` for truth checks

Replace with `is_trading_action("buy")`

(E712)